ci/clone_and_test.ps1

```diff
@@ -20,8 +20,9 @@ cd C:\go_workspace\bin
 go build github.com/codilime/contrail-windows-docker
 
 echo "Downloading test framework"
-go get github.com/onsi/ginkgo/ginkgo
-go get github.com/onsi/gomega
+go get -u github.com/onsi/ginkgo/ginkgo
```

Comment:
This is just test framework, not some dependency. All deps are frozen in vendors dir anyways.